### PR TITLE
chore(package): update ember-cli to version 2.15.1

### PR DESCRIPTION
**Pull request by simonihmig**
Closes #151

### CODE REVIEW

**Comment by kellyselden** [comment]
You could try out https://github.com/kellyselden/ember-cli-update if you want to bring the blueprint updates with you.

**Comment by simonihmig** [comment]
@kellyselden I tried that, but soon realized that it probably does not make that much sense in this case, as `e-c-a-t` itself is not an addon, just a lib, so the diff was obviously huge and imho not worth the trouble. But will give this a try for the next update of a real addon/app, thanks for the reminder! ;)

Appveyor in node 8 has a timeout error, not sure if this is just a bit flakey. Seems I have no rights to trigger a rerun, do you have?

**Comment by kellyselden** [comment]
Ah you're right. I forgot this isn't an add-on.

I don't have permission either. You could rewrite the commit and push again.

File: .gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
 node_modules
+.idea
```
Comment (kellyselden) [inline .gitignore]: This should probably go in your system .gitignore

Comment (simonihmig) [reply .gitignore]: makes sense!

**Pull request merged**